src/view/dataElements/xdmObject/components/nodeEdit.jsx

```diff
@@ -78,6 +86,9 @@ const NodeEdit = props => {
           ))}
         </Breadcrumbs>
       </View>
+      <Heading data-test-id="heading" size="M">
+        {displayName}
+      </Heading>
```

Comment:
Note that this heading is new and is showing the "display name". This wasn't really necessary, but I got carried away and included it anyway.

Reply:
I like it!